server/index.js

```diff
@@ -1,8 +1,11 @@
-var bodyParser = require('body-parser');
-var express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
I changed the variables to var because it seemed like we were using ES5 in our server/index.js, since babel doesn't transpile it. Is this incorrect?